src/core/vaults/CVault.sol

```diff
@@ -15,6 +15,12 @@ contract CVault is VaultBase {
     /// @dev Version of CVault implementation
     string public constant VERSION = '1.0.0';
 
+    /// @inheritdoc IVault
+    uint public constant UNIQUE_INIT_ADDRESSES = 1;
```

Comment:
use internal constant and public getter with overriding of vault base empty getter